src/Entity/LearningMaterial.php

```diff
@@ -677,9 +677,9 @@ public function getSessions()
      */
     public function getValidationGroups()
     {
-        if ('' !== trim($this->getCitation())) {
+        if ($this->getCitation() && strlen(trim($this->getCitation())) > 0) {
             return ['Default', 'citation'];
-        } elseif ('' !== trim($this->getLink())) {
+        } elseif ($this->getLink() && strlen(trim($this->getLink())) > 0) {
```

Comment:
see comment above, should be: 

`} elseif (null !== $this->getLink() && strlen(trim($this->getLink())) > 0) {`

Reply:
👍